process-automation/latest/admin/monitor.md

```diff
@@ -174,3 +174,4 @@ The properties for each audit event are:
 | Event ID | The unique ID of the event. |
 | Process Instance ID | The process instance ID of the process that the event occurred in. |
 | Event Action | The payload of the event in JSON format. Click to view the details. |
+| Triggers | Whenever a trigger is matched the event is added to the audit table. |
```

Comment:
You are describing this table in the admin app 
![image](https://user-images.githubusercontent.com/7249774/118255219-46cac300-b4ac-11eb-9613-72d28b71549e.png)

That trigger column is not there, so I think you should drop this addition

Reply:
Done thanks